section4/exercises/classMethods.js

```diff
@@ -28,13 +28,30 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
+changeProtein() {
+  this.protein = "chicken";
+}
+
+  addTopping(toppings) {
+    this.toppings.push(toppings);
```

Comment:
Nit pick here, but I would name the param `topping`, since you will just be pushing one topping at a time into your toppings array.

Reply:
Way to use that param to create a dynamic method, though!